composer.json

```diff
@@ -16,17 +16,28 @@
     "issues": "https://github.com/mehrwert/TYPO3-FAL-Quota/issues",
     "source": "https://github.com/mehrwert/TYPO3-FAL-Quota"
   },
+  "config": {
+    "optimize-autoloader": true,
+    "sort-packages": true,
+    "vendor-dir": ".build/vendor",
+    "bin-dir": ".build/bin"
+  },
   "require": {
     "php": ">=7.2,<7.3",
     "typo3/cms-backend": ">=9.5.0,<=10.4.99",
     "typo3/cms-core": ">=9.5.0,<=10.4.99",
     "typo3/cms-extbase": ">=9.5.0,<=10.4.99",
     "typo3/cms-extensionmanager": ">=9.5.0,<=10.4.99",
-    "typo3/cms-fluid": ">=9.5.0,<=10.4.99"
+    "typo3/cms-filelist": ">=9.5.0,<=10.4.99",
+    "typo3/cms-fluid": ">=9.5.0,<=10.4.99",
+    "typo3/cms-scheduler": ">=9.5.0,<=10.4.99",
+    "typo3fluid/fluid": "^2.7"
   },
   "require-dev": {
     "friendsofphp/php-cs-fixer": "^2.14",
-    "roave/security-advisories": "dev-master"
+    "roave/security-advisories": "dev-master",
+    "typo3/cms-frontend": ">=9.5.0,<=10.4.99",
+    "typo3/cms-install": ">=9.5.0,<=10.4.99"
```

Comment:
I added `typo3/cms-install` and `typo3/cms-frontend` as dev dependencies for ease of use during local development.

`typo3/cms-filelist`, `typo3/cms-scheduler`, `typo3/cms-fluid` and `typo3fluid/fluid` are added as regular dependencies, as we need this provide the FAL Quota features.

Anything else is kept as is.